cve_bin_tool/util.py

```diff
@@ -7,37 +7,41 @@
 import sys
 from collections import defaultdict
 from enum import Enum
-from typing import NamedTuple
+from typing import List, NamedTuple, Optional, Set, Tuple, Union
```

Comment:
After all my proposed changes below this line should look like this:
```suggestion
from typing import DefaultDict, Iterator, List, NamedTuple, Pattern, Set, Union
```